src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -1234,7 +1234,45 @@ class npc_scarlet_miner : public CreatureScript
         }
 };
 
-// npc 28912 quest 17217 boss 29001 mob 29007 go 191092
+enum StableMasterRepo
+{
+    SPELL_REPO = 52265
+};
+
+class spell_stable_master_repo : public SpellScriptLoader
+{
+public:
+    spell_stable_master_repo() : SpellScriptLoader("spell_stable_master_repo") { }
+
+    class spell_stable_master_repo_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_stable_master_repo_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_REPO });
+        }
+
+        void HandleDummy(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+        {
+            Creature* target = GetTarget()->ToCreature();
```

Comment:
If in any way this spell will be casted on player, worldserver will crash due to this part :P

Reply:
```
Attributes: 0x21000000 (SPELL_ATTR0_CASTABLE_WHILE_MOUNTED, SPELL_ATTR0_UNAFFECTED_BY_INVULNERABILITY)
AttributesEx2: 0x00000004 (SPELL_ATTR2_CAN_TARGET_NOT_IN_LOS)
AttributesEx3: 0x10000000 (SPELL_ATTR3_UNK28)
AttributesEx5: 0x00000008 (SPELL_ATTR5_USABLE_WHILE_STUNNED)
AttributesEx6: 0x00001000 (SPELL_ATTR6_CASTABLE_WHILE_ON_VEHICLE)
````
agreed, this spell needs TYPEID validation at least

Reply:
Sorry, forgot to mention - conditions are applied in #20767.

Still better to avoid any chance of crash :P yep.

Reply:
Ah makes more sense now :P